Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM php:apache
+MAINTAINER David A. Lareo "dalareo@gmail.com"
+
+RUN apt-get update -y \
+	&& apt-get install -y git libmcrypt-dev libldap2-dev \
+	&& docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu/ \
+
+	&& docker-php-ext-install mcrypt ldap
+
+RUN git clone https://github.com/simplesamlphp/simplesamlphp.git /var/simplesamlphp
```

Comment:
At this point, you have already cloned the repo, so... is it really necessary to clone it again? Wouldn't it make more sense to just make the current dir available to the container, so that it can also be used by developers to get a development version up and running quickly, where they can develop locally and test in the container?